src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -159,8 +159,15 @@ public int Indent_Braces
 
         public int Indent_CaseContents
         {
-            get { return GetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection); }
-            set { SetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection, value); }
+            get
+            {
+                return (int)_workspace.Options.GetOption(CSharpFormattingOptions.IndentSwitchCaseSection);
```

Comment:
If I rewrite to keep the original option the same and add a new option for the tri-state, and map the original against the new, then this should resolve the concern here.